package.json

```diff
@@ -4,12 +4,12 @@
   "description": "Documentation of Agoric products",
   "type": "module",
   "scripts": {
-    "docs:dev": "NODE_OPTIONS=--openssl-legacy-provider vuepress dev main",
-    "docs:build": "NODE_OPTIONS=--openssl-legacy-provider vuepress build main",
-    "docs:build-cf": "NODE_OPTIONS=--openssl-legacy-provider vuepress build main && cp _redirects main/.vuepress/dist/",
-    "docs:build-root": "NODE_OPTIONS=--openssl-legacy-provider yarn docs:build && yarn docs:re-root && cp _redirects main/.vuepress/dist-root/",
-    "docs:re-root": "cd main && mkdir -p .vuepress/dist-root/ && cp -rp .vuepress/dist/ .vuepress/dist-root/documentation/",
-    "check-links": "NODE_OPTIONS=--openssl-legacy-provider vuepress check-md main",
```

Comment:
what happened to `check-links`?

**important**

Reply:
`check-links` is noted in `CONTRIBUTING.md`. I wonder if this PR is consistent with the rest of the stuff in there.

Reply:
`check-links` logic is still there - it's natively part of of vitepress via [ignoreDeadLinks](https://vitepress.dev/reference/site-config#ignoredeadlinks), configured [here](https://github.com/Agoric/documentation/pull/971/files#diff-b1d4699f86a0fedcc35165da40f09509f3720eefa1cbec359a94b635b72575aaR137-R140), and will run during `yarn docs:build`.

Example failure:
```bash
yarn docs:build
yarn run v1.22.5
$ NODE_OPTIONS=--openssl-legacy-provider vitepress build main

  vitepress v1.0.0-rc.42

(!) Found dead link ./../js-fprogramming/index in file guides/zoe/contract-upgrade.md

If this is expected, you can disable this check via config. Refer: https://vitepress.dev/reference/site-config#ignoredeadlinks

✖ building client + server bundles...
build error:
Error: [vitepress] 1 dead link(s) found.
```

CONTRIBUTING.md changes are on the radar, added two more commits [6320811](https://github.com/Agoric/documentation/pull/971/commits/6320811deb795d6add9f919aa3277dd965b85509), [5594baa](https://github.com/Agoric/documentation/pull/971/commits/5594baa524d2d9c083cb5db66a15fbcc820317c6) that adjustments.